test/defaults.jl

```diff
@@ -0,0 +1,252 @@
+using AbstractDifferentiation
+using Test
+using FiniteDifferences, ForwardDiff, Zygote
+
+include("test_utils.jl")
+
+const FDM = FiniteDifferences
+
+## FiniteDifferences
+struct FDMBackend1{A} <: AD.AbstractFiniteDifference
+    alg::A
+end
+FDMBackend1() = FDMBackend1(central_fdm(5, 1))
+const fdm_backend1 = FDMBackend1()
+# Minimal interface
+AD.@primitive function jacobian(ab::FDMBackend1, f, xs...)
+    return FDM.jacobian(ab.alg, f, xs...)
+end
+
+struct FDMBackend2{A} <: AD.AbstractFiniteDifference
+    alg::A
+end
+FDMBackend2() = FDMBackend2(central_fdm(5, 1))
+const fdm_backend2 = FDMBackend2()
+AD.@primitive function pushforward_function(ab::FDMBackend2, f, xs...)
+    return function (vs)
+        FDM.jvp(ab.alg, f, tuple.(xs, vs)...)
```

Comment:
probably, we should add here the conversion to a tuple? (To avoid the conversion in the tests)

Reply:
Done! Seems to have done the trick.

Reply:
agreed